modules/serverless-kafka-sink.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-kafka-developer.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-sink_{context}"]
+= Using a Kafka sink
+
+This section describes how to create and configure the use of a Kafka sink by using YAML files.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource (CR) are installed on your cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have access to a Red Hat AMQ Streams (Kafka) cluster that produces the Kafka messages you want to import.
+* You have installed the `oc` CLI.
+
+.Procedure
+
+. Create a `KafkaSink` object as a YAML file:
++
+.Kafka sink YAML
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1alpha1
+kind: KafkaSink
+metadata:
+  name: <sink-name>
+  namespace: <namespace>
+spec:
+  topic: <topic-name>
+  bootstrapServers:
+   - <bootstrap-server>
+----
+
+. Apply the `KafkaSink` YAML file:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Configure an event source so that the sink is specified in its spec:
++
+.Example of a Kafka sink connected to an API server source
+[source,yaml]
+----
+apiVersion: sources.knative.dev/v1alpha2
+kind: ApiServerSource
+metadata:
+  name: <source-name>
+  namespace: <namespace>
+spec:
+  serviceAccountName: <service-account-name>
+  mode: Resource
+  resources:
+  - apiVersion: v1
+    kind: Event
+  sink:
```

Comment:
If you have the info, it would be helpful to describe the values that a user will need to customize around here. (I assume that they have to add/include this stanza and specify the `sink-name` on line 63 that they defined in the YAML file.)

Reply:
Added some callouts to clarify these, thanks